docs/using-gitbase/configuration.md

```diff
@@ -16,6 +16,7 @@
 | `GITBASE_CACHESIZE_MB`       | size of the cache for git objects specified as MB                                  |
 | `GITBASE_CONNECTION_TIMEOUT` | timeout in seconds used for client connections on write and reads. No timeout by default.     |
 | `GITBASE_USER_FILE`          | JSON file with user credentials                                                    |
+| `GITBASE_MAX_UAST_BLOB_SIZE`          | Max size of blobs to send to be parsed by bblfsh. No limit by default.                                                    |
```

Comment:
I have generic question - do people really know (should know) anything about bblfsh and/or UAST, when using gitbase?

Reply:
I guess not using gitbase, but yes when using uasts in gitbase